ci-operator/step-registry/storage/conf/csi-aws-ebs/storage-conf-csi-aws-ebs-commands.sh

```
@@ -7,12 +7,8 @@ if [ -d /go/src/github.com/openshift/csi-operator/ ]; then
     echo "Using csi-operator repo"
     cd /go/src/github.com/openshift/csi-operator
     cp test/e2e/aws-ebs/manifest.yaml ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
-    if [ -n "${TEST_OCP_CSI_DRIVER_MANIFEST}" ]; then
-        if [ "${ENABLE_LONG_CSI_CERTIFICATION_TESTS}" = "true" ]; then
-            cp test/e2e/aws-ebs/ocp-manifest-long.yaml ${SHARED_DIR}/${TEST_OCP_CSI_DRIVER_MANIFEST}
-        else
-            cp test/e2e/aws-ebs/ocp-manifest.yaml ${SHARED_DIR}/${TEST_OCP_CSI_DRIVER_MANIFEST}
-        fi
+    if [ -n "${TEST_OCP_CSI_DRIVER_MANIFEST}" ] && [ "${ENABLE_LONG_CSI_CERTIFICATION_TESTS}" = "true" ]; then
+        cp test/e2e/aws-ebs/ocp-manifest.yaml ${SHARED_DIR}/${TEST_OCP_CSI_DRIVER_MANIFEST}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)